test/MOI/MOI_multiobjective.jl

```diff
@@ -55,7 +55,7 @@ c4: y >= 0.25
     @test MOI.get(model, Gurobi.MultiObjectivePriority(2)) == 0
 
     MOI.optimize!(model)
-    @test MOI.get(model, MOI.ObjectiveValue()) ≈ 1.5
+    @test MOI.get(model, MOI.ObjectiveValue()) ≈ [1.5, 2.0]
```

Comment:
**This is a breaking change** (albeit, a minor one for someone using the multi-objective interface to Gurobi)

Reply:
@blegat: this is a good enough time as any to release a Gurobi v1.0.0?

Reply:
I don't think we plan any more breaking changes so I don't see any objection

Reply:
Okay. I'll merge and then make a few other changes before tagging Gurobi v1.0